habitat/omnitruck-poller/hooks/run

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+exec 2>&1
+
+. /hab/svc/omnitruck/config/dotenv
+
+cd $OMNITRUCK_HOME
+
+exec bundle exec ./poller -e production
+
+# TODO: Figure out why sleep ends script
```

Comment:
I tried that too, but it still needs to sleep. Same result. The poller script could be enhanced to loop, but I'd prefer to have the run hook handle it.